main.go

```diff
@@ -25,8 +26,23 @@ func main() {
 	copyDirFlag := flag.String("copy-dir", "", "Path to a folder where HIE records should be copied locally (env: COPY_DIR, default: none)")
 	cronFlag := flag.String("cron", "", "Cron expression indicating when the integrator tool should run to refresh data (env: INTEGRATOR_CRON, example: \"0 0 20 * * *\").  If cron is not supplied, \"now\" must be set.")
 	nowFlag := flag.Bool("now", false, "Flag to indicate if the integrator should run immediately (env: INTEGRATOR_NOW, default: false).  If used without cron, integrator will run once and then exit.  If now is not set, \"cron\" must be supplied.")
+	logFileFlag := flag.String("logdir", "", "Path to a directory for integrator logs to be written to.")
 	flag.Parse()
 
+	lfpath := getConfigValue(logFileFlag, "INTEGRATOR_LOG_DIR", "")
+	if lfpath != "" {
+		err := os.Mkdir(lfpath, 0755)
+		if err != nil && !os.IsExist(err){
+			fmt.Println("Error creating log directory:" + err.Error())
+		}
+		lf, err := os.OpenFile(lfpath + "/integrator.log", os.O_RDWR|os.O_APPEND, 0755)
+		if os.IsNotExist(err) {
+			lf, err := os.Create(lfpath + "/integrator.log")
+		} else if err != nil {
```

Comment:
This needs to be a new `if` statement instead of an `if else`.  Otherwise the `else` (`log.SetOutput(lf)`) won't get called when `os.IsNotExist` (since it's all part of the same big if branch).